index.md

```diff
@@ -26,22 +26,33 @@ redirect_from:
+Zowe is an integrated and extensible open source framework for z/OS. Zowe, like Mac OS or Windows, comes with a set of APIs and OS capabilities that applications build on and also includes some applications out of the box.
 </p>
 
 <p>
-Zowe offers modern interfaces to interact with z/OS and allows you to work with z/OS in a way that is similar to what you experience on cloud platforms today. You can use these interfaces as delivered or through plug-ins and extensions that are created by clients or third-party vendors.
+Zowe offers modern interfaces to interact with z/OS and allows you to work with z/OS in a way that is similar to what you experience on cloud platforms today. You can use these interfaces as delivered or through plug-ins and extensions that are created by clients or third-party vendors. 
 </p>
 
-<p>Zowe consists of the following main components.</p>
+<p>Zowe consists of the following components.</p>
 
 <p><b>Zowe Application Framework:</b> A web user interface (UI) that provides a virtual desktop containing a number of apps allowing access to z/OS function.  Base Zowe includes apps for traditional access such as a 3270 terminal and a VT Terminal, as well as an editor and explorers for working with JES, MVS Data Sets and Unix System Services.</p>
 
 <p><b>API Mediation Layer:</b> Provides a gateway that acts as a reverse proxy for z/OS services, together with a catalog of REST APIs and a dynamic discovery capability. Base Zowe provides core services for working with MVS Data Sets, JES, as well as working with z/OSMF REST APIs. The API Mediation Layer also provides a framework for Single Sign On (SSO). </p>
 
 <p><b>Zowe CLI:</b> Provides a command-line interface that lets you interact with the mainframe remotely and use common tools such as Integrated Development Environments (IDEs), shell commands, bash scripts, and build tools for mainframe development. It provides a set of utilities and services for application developers that want to become efficient in supporting and building z/OS applications quickly. The CLI provides a core set of commands for working with data sets, USS, JES, as well as issuing TSO and console commands.</p>
 
-<p><b>Zowe Client Software Development Kits (SDKs):</b> Provides access to a set of programmatic APIs that you can use to build client applications or scripts that interact with z/OS. SDKs are available for Node.js and Python.</p>
+<p><b>Zowe Explorer:</b> A Visual Studio Code extension that lets you interact with data sets, USS files and jobs that are stored on z/OS. Working with data sets and USS files from VS Code can be more convenient than using 3270 emulators, and complements your Zowe CLI experience.
```

Comment:
```suggestion
<p><b>Zowe Explorer:</b> A Visual Studio Code extension that lets you interact with data sets, USS files, and jobs that are stored on z/OS. Working with data sets and USS files from VS Code can be more convenient than using 3270 emulators, and complements your Zowe CLI experience.
```
Just that oxford comma :) Otherwise, looks good!